packages/web-components/fast-components/src/menu-item/menu-item.styles.ts

```diff
@@ -0,0 +1,78 @@
+import { css } from "@microsoft/fast-element";
+import {
+    disabledCursor,
+    display,
+    focusVisible,
+    heightNumber,
+    neutralFillStealthActiveBehavior,
+    neutralFillStealthHoverBehavior,
+    neutralFillStealthRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundRestBehavior,
+} from "../styles";
+
+export const MenuItemStyles = css`
+    ${display("grid")} :host {
+        outline: none;
+        box-sizing: border-box;
+        height: calc(${heightNumber} * 1px);
+        ${
+            /* grid-template-columns:  calc(var(--design-unit) * 1px) auto calc(var(--design-unit) * 1px); */ ""
```

Comment:
What's the purpose of leaving this commented out? If there is one, can we add a TODO or something?

Reply:
Removing for now.